test_common.go

```diff
@@ -36,6 +36,13 @@ var (
   "y": "4Etl6SRW2YiLUrN5vfvVHuhp7x8PxltmWWlbbM4IFyM",
   "d": "870MB6gfuTJ4HtUnUvYMyJpr5eUZNP4Bk43bVdj3eAE"
 }`
+	TFMECKey = `{
```

Comment:
The indentation looks a bit off to me? Please check!

Reply:
LGTM

Reply:
checked, it's ok